src/components/Pages/workshopOverview/children/FeedbackSection.js

```diff
@@ -26,7 +26,12 @@ export default function Comments({ feedbackArr }) {
 	}, [feedbackArr]);
 
 	function CreateCommentsList(feedbackNewArray) {
-		return feedbackNewArray.map((commentObj) => <FeedbackListItem commentObj={commentObj} />);
+		return feedbackNewArray.reverse().map((commentObj) => {
```

Comment:
Cheeky reverse!

Reply:
you know 😂

Reply:
I mean we should be using date created and auto generating it when the post request is made